client/src/transmitter/types.d.ts

```diff
@@ -0,0 +1,10 @@
+export {}
+
+declare global {
+  interface CastSession {
+    stop(): void
+
```

Comment:
Remove `"\n"`.

Reply:
It was to separate the "constant" from the "mutable" properties, since there's no clear way of expressing/enforcing that. But no prob removed it, they're currently 'separate' by being declared either as a function or a property which is a function.